components/uid-input/stories/index.stories.ts

```diff
@@ -58,3 +89,18 @@ SlottedContent.args = {
 SlottedContent.argTypes = {
   slot: { table: { disable: true } },
 };
+
+function getLabelHidden(labelHidden: boolean) {
+  if (!labelHidden) {
```

Comment:
:x: didn't work like other boolean attributes

Reply:
will be done by @jeromecambon in another PR